esmvaltool/cmorizers/obs/cmorize_obs_noaa_ersstv3b.py

```diff
@@ -0,0 +1,69 @@
+"""ESMValTool CMORizer for NOAA Extended Reconstructed Sea Surface
+   Temperature (ERSST), Version 3b data.
+
+Tier
+    Tier 2: open dataset.
+
+Source
+    https://doi.org/10.1175/1520-0442-16.10.1495
+
+Last access
+    20200520
+
+Download and processing instructions
+    TODO
+
```

Comment:
yr=1854
while [ $yr -le 2016 ]; do
nm=1
while [ $nm -le 12 ]; do
if [ $nm -lt 10 ]; then
nm=0$nm
fi
wget **ftp://ftp.ncdc.noaa.gov/pub/data/cmb/ersst/v5/netcdf/ersst.v5.$yr$nm.nc**
nm=`expr $nm + 1`
done
yr=`expr $yr + 1`
done

Reply:
I added your info to the comment.